### PR TITLE
Adding Demo UI to Aperture Playground

### PR DESCRIPTION
**Pull request by gurinder39**
### Description of change

- Added a static app to demonstarate mutliple scenario 

### Summary by OpenAI

**New Feature:**
- Added a demo app to demonstrate multiple scenarios
- Introduced new React components `MonitorRequest`, `HomePage`, and `DemoAppThemeProvider`
- Implemented rate-limited handler function in the backend
- Created an axios instance with a base URL and headers

**Chore:**
- Added configuration files for ESLint, Prettier, and Webpack
- Updated `.gitignore` file
- Included license information for third-party libraries

> 🎉 A demo app to showcase our might,
> New components shining bright,
> Rate limits handled with care,
> Configurations added everywhere! 🚀

### Summary by CodeRabbit

**New Features:**
- Introduced `demoui` namespace and a static app for demonstrating multiple scenarios.
- Added new React components and helper functions.
- Implemented rate-limited handler function in the backend.
- Created an axios instance with a base URL and headers.
- Updated configuration files for ESLint, Prettier, and Webpack.
- Added Dockerfiles to build and run the application.

**Chore:**
- Updated `.gitignore` file.
- Added license information for third-party libraries.
- Included a k6 load testing script.

> 🎉 A new dawn breaks in our codebase, bright and clear, 🌅
> With namespaces and handlers, we have nothing to fear. 💪
> Axios and React, together they unite, 🤝
> Making our app's performance take flight. 🚀
> Configurations updated, Dockerfiles added too, 🐳
> Celebrate these changes, for our code is renewed! 🎊

### CODE REVIEW

**Review by karansohi — APPROVED** [review]

File: playground/Tiltfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ APERTURE_JS_EXAMPLE_NS = "aperture-js-example"
 APERTURE_JAVA_EXAMPLE_NS = "aperture-java-example"
 APERTURE_PYTHON_EXAMPLE_NS = "aperture-python-example"
 APERTURE_DEMOAPP_NS = "demoapp"
+APERTURE_DEMO_UI_NS = "demoui"
```
Comment (hasit) [inline playground/Tiltfile:43]: We can keep it in `demoapp` namesapce.

File: playground/resources/demo-ui/src/pages/home.tsx [hunk 2]
```diff
@@ -4,61 +4,126 @@ import {
...
+  } = makeRequestToEndpoint(reqSpec)
+
+  // Request to workload-prioritization endpoint with Subscriber user
+  reqSpec.userType = 'Subscriber'
+  const {
+    refetch: refetchSubscriber2,
+    isError: isErrorSubscriber2,
+    requestRecord: subscriberRequestRecord2,
+    isLoading: isLoadingSubscriber2,
+  } = makeRequestToEndpoint(reqSpec)
```
Comment (gurinder39) [inline playground/resources/demo-ui/src/pages/home.tsx]: @karansohi follow react hook convenstion. If it is a hook add use infront of that function name `useRequestToEndpoint`

File: playground/resources/demo-ui/src/pages/home.tsx [hunk 3]
```diff
@@ -4,61 +4,126 @@ import {
...
-        }}
-        isErrored={isErrorSubscriber}
-        isLoading={isLoadingSubscriber}
-        errorComponentProps={{
-          url: '/api/rate-limit',
-          requestBody: {},
-        }}
-      />
-    </>
+    <TabContext value={value}>
```
Comment (gurinder39) [inline playground/resources/demo-ui/src/pages/home.tsx:62]: To add tabs you can follow: [link](https://github.com/fluxninja/cloud/blob/main/services/cloud/ui/src/apps/organization/pages/authenticated/alert-manager/page.tsx)

**Review by karansohi — APPROVED** [review]

**Comment by karansohi** [comment]
image
image
image

**Pull request merged**